ROADMAP.md

```diff
@@ -1,13 +1,13 @@
 # Roadmap
 
-## Q2.2024
+## v0.5
 
-* [ ] Untyped API
-* [x] C binding
+* [ ] (Waitset), event multiplexing and advanced push notifications
+* [ ] finalize C and C++ API
+* [ ] Fully dynamic payload for pub/sub
 * [ ] Derive macro for SHM transferable types
-* [ ] Node as basis for monitoring and resource cleanup
 * [ ] `serde`-based shm serialization to transmit arbitrary types
-* [ ] Pipeline (optional Request/Response)
+* [ ] Request/Response
```

Comment:
What do you think about splitting this into something like
```
## v0.5

### core (for the lack of a better word)
* [ ] foo

### stretch
* [ ] bar
```

Everything from `stretch` would be promoted to `core` of the next release if not implemented in the current release.